Jenkinsfile

```
@@ -17,10 +17,27 @@ pipeline {
 
     stages {
         stage('Build Docker Image') {
-            steps {
-                deleteDir()
-                checkout scm
-                sh 'make build'
+            parallel {
+                stage('Windows') {
+                    agent {
+                        label "windock&&windows"
```

```suggestion
                        label 'windock'
```

would presumably suffice?